osv/purl_helpers.py

```diff
@@ -45,4 +46,7 @@ def package_to_purl(ecosystem, package_name):
     # PURLs use / to separate the group ID and the artifact ID.
     package_name = package_name.replace(':', '/', 1)
 
+  if purl_type == 'deb':
+    package_name = 'debian/' + package_name
```

Comment:
Should we add `?arch=source` ?

Reply:
It probably should? However the way purl is put into the database is just as a string, and queried by identical string matching, to properly match parameters will need more logic.

Reply:
That's a good point, but we should solve that separately -- possibly by just adding an index which includes the common permutations we'd expect (either no parameters or with "arch=").